appinventor/appengine/src/com/google/appinventor/client/GalleryGuiFactory.java

```diff
@@ -181,6 +181,103 @@ public void onClick(ClickEvent event) {
   }
 
   /**
+   * Loads the proper tab GUI with gallery's app data.
+   * @param apps: list of returned gallery apps from callback.
+   * @param container: the GUI panel where apps will reside.
+   * @param refreshable: if true then the GUI can be reloaded later.
+   */
+  public void generateFeaturedAppList(List<GalleryApp> apps,
+                                        FlowPanel container, Boolean refreshable) {
+    if (refreshable) {
+      // Flush the panel's content if we knew new stuff is coming in!
+      container.clear();
+    }
+    for (final GalleryApp app : apps) {
+      // Create the associated GUI object for app
```

Comment:
There are a number of text comments that could be deleted, such as this line, it doesn't really add much.

There's also a number of code lines commented out that should also go. Two in particular are override annotations; is the code overriding or not?